### PR TITLE
don't depend on symfony 2.7 for graviton

### PR DESCRIPTION
**Pull request by hairmare**
This also bumps all the remaining things and makes sure that tests
run by switching to lapistano/proxy-object@master.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,13 +14,13 @@
...
         "php": "~5.4",
-        "symfony/http-kernel": "~2.7",
+        "symfony/http-kernel": "~2.6",
         "graviton/php-rql-parser": "~2.0@alpha"
     },
 
     "require-dev": {
         "symfony/framework-bundle": "~2.7",
-        "lapistano/proxy-object": "~2.1",
+        "lapistano/proxy-object": "dev-master",
```
Comment (narcoticfresh) [inline composer.json]: why depend on `master` and tags?

Comment (hairmare) [inline composer.json]: due to https://github.com/lapistano/proxy-object/pull/11...

Comment (narcoticfresh) [inline composer.json]: Hm.. so do we only temporary depend on it like that to kinda test the change? Shouldn't @lapistano make a new release? ;-)

Comment (hairmare) [inline composer.json]: It's only in ``"require-dev"`` and I want to finish this asap...

**Pull request merged**